src/com/crdroid/updater/UpdatesActivity.java

```diff
@@ -135,9 +137,160 @@ public void onReceive(Context context, Intent intent) {
                 getString(R.string.header_android_version, Build.VERSION.RELEASE));
 
         TextView headerBuildDate = (TextView) findViewById(R.id.header_build_date);
-        headerBuildDate.setText(StringGenerator.getDateLocalizedUTC(this,
+        headerBuildDate.setText("Current build date: " + StringGenerator.getDateLocalizedUTC(this,
```

Comment:
why hard code?

Reply:
Because we don't translate this repo anyway

Reply:
we might move it to crdroid settings.

Reply:
Can be done afterwards then